.github/workflows/download-trials-release.yml

```
@@ -0,0 +1,56 @@
+---
+name: Release Download trials
+
+on:
+  workflow_dispatch:
+
+env:
+  DEFAULT_BRANCH_NAME: master
+  TRIALS_BRANCH_NAME: download-trial
+
+jobs:
+  createPR:
```

We use snake_case in other workflows
```suggestion
  create_pr:
```